.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1' # automatically expands to the latest stable 1.x release of Julia
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+          - x86
+        exclude:
```

Comment:
Do we actually need to test on all combinations of julia version, os and architecture?

Reply:
Yeah that's a difficult point. AFAIK we used to do that until now. Another solution would be to do something like https://github.com/JuliaLang/Example.jl/pull/55. But given that StatsBase has a relatively low activity it may be OK to run many checks.

Reply:
Yeah, that's what we do. If you don't think StatsBase is gonna be that active going forward then I'm fine with leaving it.